src/CMakeLists.txt

```diff
@@ -838,16 +831,35 @@ optional_source(HAVE_LIBLASTFM
     internet/lastfm/lastfmsettingspage.ui
 )
 
-
-optional_source(HAVE_SPOTIFY_DOWNLOADER
+# Spotify support
+optional_source(HAVE_SPOTIFY
   SOURCES
-    internet/spotify/spotifyblobdownloader.cpp
+    internet/spotify/spotifyserver.cpp
+    internet/spotify/spotifyservice.cpp
+    internet/spotify/spotifysettingspage.cpp
+    globalsearch/spotifysearchprovider.cpp
+    songinfo/spotifyimages.cpp
```

Comment:
This shouldn't be behind the Spotify flag - it's not really related to the other Spotify support.